src/test/java/com/linecorp/armeria/client/http/HttpClientIntegrationTest.java

```diff
@@ -127,6 +132,16 @@ protected void doGet(ServiceRequestContext ctx, HttpRequest req,
                     res.respond(HttpStatus.NOT_FOUND);
                 }
             });
+
+            sb.serviceAt("/useragent", new AbstractHttpService() {
+                @Override
+                protected void doGet(ServiceRequestContext ctx, HttpRequest req,
+                                     HttpResponseWriter res) {
```

Comment:
These two lines could be joined into one because it will not exceed 112 columns.